src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -28,14 +34,49 @@
      * 26.04.2019 Сергей 7 100
      * 26.04.2019 София 9 100
      * 26.04.2019 Сергей 11 50
-     *
+     * </p>
      * Пример вывода:
      * Отчёт за период 01.04.2019  - 30.04.2019
      * Сергей - 1550
      * Андрей - 600
      * София - 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder allSalary = new StringBuilder();
+        allSalary.append("Отчёт за период " + dateFrom + " - " + dateTo + "\n");
+        DateFormat format = new SimpleDateFormat("dd.MM.yyyy", Locale.ENGLISH);
+        int salary1 = 0;
```

Comment:
@Valdes2 this is hardcode. What if you will need to calculate salaries for 5 or 6 persons?

Reply:
done